.github/workflows/tests-saas.yml

```diff
@@ -17,6 +17,21 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Free disk space
```

Comment:
Do we care about the disk space in saas test?

Reply:
Yes, as I wrote, I needed to free disk space as the CI failed with "no space left on device".